charts/kserve-resources/templates/clusterservingruntimes.yaml

```diff
@@ -475,7 +479,56 @@ spec:
         sizeLimit: {{ . }}
         {{- end }}
   {{- end }}
-  {{- with .Values.kserve.servingruntime.huggingfaceserver.imagePullSecrets }}
-  imagePullSecrets:
-    {{- toYaml . | nindent 4 }}
+---
+apiVersion: serving.kserve.io/v1alpha1
+kind: ClusterServingRuntime
+metadata:
+  name: kserve-huggingface-triton-server
+spec:
+  annotations:
+    prometheus.kserve.io/port: '8080'
+    prometheus.kserve.io/path: "/metrics"
+  supportedModelFormats:
+    - name: triton-hf
+      version: "1"
+      autoSelect: true
+      priority: 1
```

Comment:
I think the model format should have the same list as triton runtime, it can be the huggingface model converted to torchscript or onnx